recipes/jasper/all/conanfile.py

```diff
@@ -39,12 +41,20 @@ def config_options(self):
             del self.options.fPIC
 
     def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
         del self.settings.compiler.cppstd
         del self.settings.compiler.libcxx
 
+        # Handle deprecated libjpeg option
+        self.output.warn("jpegturbo option is deprecated, use with_libjpeg option instead.")
```

Comment:
This will always print, maybe it should only be shown when it's `True` ?

Reply:
Yes maybe.